h5p-bildetema/src/components/SearchPage/SearchFilter/SearchFilterDialog.module.scss

```
@@ -0,0 +1,92 @@
+@use "common/abstracts" as *;
+
+.dialog {
+  z-index: 50;
+  position: relative;
+}
+.backdrop {
+  position: fixed;
+  inset: 0;
+  background-color: rgba(0, 0, 0, 0.3);
+}
+
+.titleWrapper {
+  font-size: $font-size-20;
+  display: flex;
+  justify-content: space-between;
+  gap: 3rem;
+  line-height: 0;
+}
+
+.title {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.panelContainer {
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  inset: 0;
+}
+
+.panel {
+  max-height: 85vh;
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  padding: 2rem 3rem;
+  border-radius: 0.5rem;
+  background-color: $beige;
+  overflow-y: auto;
+}
+
+.searchFilter {
+  display: block;
+  column-count: 1;
+  gap: 5rem;
+  font-size: $font-size-20;
+  line-height: 1;
+
+  @include breakpoint-min($x-small) {
+    column-count: 1;
+  }
+
+  @include breakpoint-min($small) {
+    column-count: 2;
+  }
+  @include breakpoint-min($medium) {
+    column-count: 3;
+  }
+  @include breakpoint-min($large) {
+    column-count: 4;
+  }
+  @include breakpoint-min($x-large) {
+    // column-count: 5;
+  }
+}
+
+.checkBoxWrapper {
+  display: flex;
+  align-items: center;
+  justify-items: end;
+  padding: 0.5rem 0;
+}
+
+.closeButton {
+  all: unset;
+  cursor: pointer;
+}
+
+.amount {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: $white;
+  width: 2rem;
+  height: 2rem;
+  border-radius: 100%;
+  background-color: $dark-teal;
```

In order to get the number to align better vertically we can set the `line-height` to be bore equal to the actual `font-size`. Use `line-height: 1;` to get the same height as the `font-size`. 

```suggestion
  background-color: $dark-teal;
  line-height: 1;
```